docs/src/gpu.md

```diff
@@ -0,0 +1,57 @@
+# GPU Support
+
+We are currently actively working on GPU support.
+As of now, the [`WeaklyCompressibleSPHSystem`](@ref) and the [`BoundarySPHSystem`](@ref)
+are supported on GPUs.
+We have tested this on GPUs by Nvidia and AMD.
+
+To run a simulation on a GPU, we need to use the [`FullGridCellList`](@ref)
+as cell list inside the [`GridNeighborhoodSearch`](@ref).
+This cell list requires a bounding box for the domain, unlike the default cell list, which
+uses an unbounded domain.
+For simulations that are bounded by a closed tank, we can use the boundary of the tank
+to obtain the bounding box as follows.
+```jldoctest gpu; output=false, setup=:(using TrixiParticles; trixi_include(@__MODULE__, joinpath(examples_dir(), "fluid", "hydrostatic_water_column_2d.jl"), sol=nothing))
+search_radius = TrixiParticles.compact_support(smoothing_kernel, smoothing_length)
+min_corner = minimum(tank.boundary.coordinates, dims=2) .- search_radius
+max_corner = maximum(tank.boundary.coordinates, dims=2) .+ search_radius
+cell_list = TrixiParticles.PointNeighbors.FullGridCellList(; min_corner, max_corner)
+
+# output
+PointNeighbors.FullGridCellList{PointNeighbors.DynamicVectorOfVectors{Int32, Matrix{Int32}, Vector{Int32}, Base.RefValue{Int32}}, Nothing, SVector{2, Float64}, SVector{2, Float64}}(Vector{Int32}[], nothing, [-0.24500000000000002, -0.24500000000000002], [1.245, 1.245])
+```
```

Comment:
Can't we default to setting this bounding box?

Reply:
Not really. What if we don't use a tank? Or if the tank doesn't have a closed top? 
I think it makes more sense to work on cell lists that support unbounded domains.

Reply:
I am not saying to remove the option to overwrite the default bounding box. But using the minimum/maximum as a default seems to me to be a reasonable choice.

Reply:
The problem is that this is an option of the neighborhood search. The neighborhood search for fluid-fluid only sees the coordinates of the fluid, so we can't set the bounding box to the maximum and minimum of the boundary coordinates.

Reply:
Yes it would need to be set in semidiscretization I guess.

Reply:
Discussion in https://github.com/trixi-framework/TrixiParticles.jl/issues/665.